### ISSUE TITLE
Full Windows paths - e.g. drive letters - cause promise rejection "TypeError: Failed to fetch"

### ISSUE THREAD

**Issue by shrewdlogarithm** [MikeKovarik/exifr#61]
I tested a few variants of this and it's not the separators but the 'drive letter:' at the start which seems to throw exifr off?

e.g.

exifr.parse("c:\pathtoapp\test.jpg") << fails
exifr.parse("c:/pathtoapp/test.jpg") << also fails
exifr.parse("./test.jpg") << obviously works

Given that fs.readfile etc. work with drive letters and they're returned by INPUT FILE tags it's a pain-in-the-ass to commute the paths - I think exifr should probably support them (I guess it's throwing-away the : perhaps?)

**Comment by MikeKovarik** [comment]
Hello. I'm sure absolute paths are supported, since it is internally passed to `fs.read()`. But just to be sure I'm trying this out right now with the following code and I can't replicate your problem.
```js
//let filePath = 'c:\\Users\\Mike\\Downloads\\IMG_6253.jpg'
let filePath = 'c:/Users/Mike/Downloads/IMG_6253.jpg'
let output = await exifr.parse(filePath).then(console.log).catch(console.error)
```
It works with both path variants (`/` as well as `\` -note the need to escape `\` with second `\`)and I also tried both uppercase and lowercase `C` and `c`.

Could you give me some more info about your environment, perhaps about the code? This is such a strange behavior, I don't even know where to look or what info to ask for :D

**Comment by MikeKovarik** [comment]
Hello. This is likely fixed by 7.0.0. I haven't tested it myself but I'm pretty confident of what was causing the problem and that it should be fixed. If not, feel free to reopen the issue

**Comment by shrewdlogarithm** [comment]
Sorry I didn't get back quicker - I have a demo to test, I'll hold-off until I can test with 7.0 and report back accordingly - thanks for looking at this!
